core/client/db/src/lib.rs

```diff
@@ -67,7 +67,7 @@ use utils::{Meta, db_err, meta_keys, open_database, read_db, read_id, read_meta}
 use state_db::StateDb;
 pub use state_db::PruningMode;
 
-const FINALIZATION_WINDOW: u64 = 32;
+const CANONICALIZATION_DELAY: u64 = 256;
```

Comment:
That's quite a lot for the current implementation that keeps uncanonicalized states in memory.  Any good reason to change it?

Reply:
32-block reorgs doesn't seem that crazy to me under the consensus algorithms we are using, especially while the testnet is unstable.